### DISCUSSION TITLE
2011-2 - 泛函分析基础：学习指导和习题详解

### DISCUSSION THREAD

**Discussion by giscus** [duskmoon314/THU_BFA#2] [Announcements]
# 2011-2 - 泛函分析基础：学习指导和习题详解

https://duskmoon314.github.io/THU_BFA/ch6/2011-2.html

**Comment by thulanxc** [comment]
第二题是不是要反过来说？就是说aA里面的任意收敛列都收敛在aA内，现在这么说的话好像还得证明axn能覆盖所有的aA中的数列。
必要性也可以直接取a=1/a。

**Reply by duskmoon314** [reply]
你说的有道理，我改改
